bundles/core/src/main/java/com/adobe/cq/email/core/components/util/StyleTokenizer.java

```diff
@@ -79,23 +81,29 @@ private StyleTokenizer() {
             }
 
             if (nestingLevel == 0) {
-                current = StyleTokenFactory.create(next);
+                current = StyleTokenFactory.create(item);
                 if (skipCheck.contains(current.getSelector())) {
                     current.setForceUsage(true);
                 }
                 current.setMediaQuery(current.getSelector().contains("@"));
-                current.setPseudoSelector(!current.isMediaQuery() && next.contains(":"));
+                current.setPseudoSelector(!current.isMediaQuery() && item.contains(":"));
                 continue;
             }
 
             if (nestingLevel > 0) {
-                if (next.contains(";") || next.contains(":")) {
-                    StyleTokenFactory.addProperties(current, next);
+                boolean isInnerPseudo = false;
+                String next = strings[index].trim();
+                if (current.isMediaQuery() && item.contains(":") && StringUtils.equals(next, "{")) {
```

Comment:
for the following css the `current.isMediaQuery()` check returns false as `current` points to the previously parsed `p` selector. I wonder if it is needed. `StringUtils.equals(next, "{")` should be sufficient to detect a next nesting level. 

```
@media screen and (max-width: 600px) {
 p { color: red; }
 a:hover { color: yellow }
}
```